youyube_search_skill.md

```
@@ -0,0 +1,273 @@
+argument-hint: "<query> [--count N]"
+allowed-tools:
+  - Bash
+---
+
+Run the YouTube search script with the user's arguments and present the results.
+
+Execute this command:
+
+```
+python ~/.claude/skills/yt-search/scripts/search.py $ARGUMENTS
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)